src/audio/dai.c

```diff
@@ -816,6 +816,7 @@ static void dai_config_reset(struct comp_dev *dev)
 static int dai_prepare(struct comp_dev *dev)
```

Comment:
Please explain WHY this patch is needed. Your commit message says WHAT the patch does. While this is also useful the commit message should first explain WHY the commit is needed and then add details about it.